examples/eks-msk-opensearch/README.md

```diff
@@ -0,0 +1,110 @@
+# Centralized logging for multi-tenant applications on Amazon EKS
+
+In this example, we will showcase how to build a centralized logging solution for multi-tenant environments where applications that belong to different teams or customers run in a shared Amazon EKS cluster.
+
+## Components
+
+[Fluent Bit](https://fluentbit.io/) DaemonSet to collect, modify, and enrich logs from applications, and publish the logs to Amazon Managed Streaming for Kafka (Amazon MSK).
+
+[Amazon MSK](https://aws.amazon.com/msk/) to forward log events to various destination and as a buffering layer to avoid indexing pressure in Amazon OpenSearch. This layer will provide tenant isolation and improve resilience of the solution.
+
+[Amazon OpenSearch](https://aws.amazon.com/what-is/opensearch/) to monitor, visualize, and analyze logs. OpenSearch is a distributed, community-driven, Apache 2.0-licensed, 100% open-source search and analytics suite used for a broad set of use cases like real-time application monitoring, log analytics, and website search.
+
+[Terraform by HashiCorp](https://www.terraform.io/), is an infrastructure as code tool similar to AWS CloudFormation to provision and manage infrastructure on AWS.
+
+## Features
+
+Security and compliance - This solution provides data isolation for logs ingested and stored so that each tenant can only access their own logs. It also can help you to meet your compliance requirements, for example anonymizing personally identifiable information (PII) in logs which is required by GDPR.
+
+Business and technical insights -  You can produce business insights such as geographical distribution of customers or popularity of a product over time. Software engineers can also use this solution to troubleshoot an issue, or to create metrics and alarms to proactively notify the application owners of any issues.
```

Comment:
Again, later it'd be great to explain how to do this- is this an opensearch dashboard feature?